airflow/example_dags/example_xcomargs.py

```diff
@@ -43,11 +44,26 @@ def print_value(value):
     default_args={'owner': 'airflow'},
     start_date=days_ago(2),
     schedule_interval=None,
-    tags=['example']
+    tags=['example'],
 ) as dag:
     task1 = PythonOperator(
         task_id='generate_value',
         python_callable=generate_value,
     )
 
     print_value(task1.output)
+
+
+with DAG(
+    "example_xcom_args_with_operators",
+    default_args={'owner': 'airflow'},
+    start_date=days_ago(2),
+    schedule_interval=None,
+    tags=['example'],
+) as dag2:
+    bash_op1 = BashOperator(task_id="c", bash_command="echo c")
+    bash_op2 = BashOperator(task_id="d", bash_command="echo c")
+    xcom_args_a = print_value("first!")
+    xcom_args_b = print_value("second!")
+
+    bash_op1 >> xcom_args_a >> xcom_args_b >> bash_op2
```

Comment:
Thanks for the quick fix. One other issue is that if we do this at this line:
```
xcom_args_a >> bash_op1 >>  xcom_args_b
```

Is this supposed to work? i have not tried yet, but looking at the code, I think `BaseOperator.__rshift__` will complain that it can't handle `xcom_args`. That's a different issue though so the fix can be done separately.

Reply:
Same DAG is used in tests to make sure that it works. `BaseOperator._set_relatives` handles XComArgs properly. However, `XComArgs` is missing in type hints due to cyclic imports. I think that once we have your PR merged we may consider something like `TaskType` that will be union of types that are "task-like"